App/ViewController.swift

```diff
@@ -35,22 +35,22 @@ class ViewController: UIViewController {
     }
 
     @IBAction func startOAuth2(_ sender: Any) {
-        var auth0 = Auth0.webAuth()
+        let auth0 = Auth0.webAuth()
```

Comment:
If you've made this non-variable you can skip the whole `let auth0 =` bit and just go straight into
```Swift
Auth0
   .webAuth()
   .logging...
```

Reply:
Yeah good point, the sample App isn't a priority. However, it will look cleaner 👍